src/api/posts/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@senecacdot/satellite": "^1.x",
+    "bull": "^3.20.1",
```

Comment:
Because you a dependency, you need to also update the `pnpm` lock file and commit that as well.